src/setup_payload/tests/TestAdditionalDataPayload.cpp

```diff
@@ -22,22 +22,20 @@
  *
  */
 
+#include <gtest/gtest.h>
```

Comment:
I would suggest to follow google c++ codestyle with include grouping:

1. standard includes
2. 3rd party includes
3. project includes

Reply:
done, thanks!

Reply:
- I took a look at the google[ C++ code style](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes), it seems that to strictly follow it I would also need to seperate the "related header" and have it before standard includes, this does not seem to be strictly enforced anyways, any thoughts?

- from my understanding, the google C++ code style will need the includes to be as below, which might overcomplicate things:

```
#include <setup_payload/AdditionalDataPayloadGenerator.h>
#include <setup_payload/AdditionalDataPayloadParser.h>
#include <setup_payload/SetupPayload.h>

#include <math.h>
#include <memory>
#include <stdio.h>

#include <gtest/gtest.h>

#include <lib/support/BytesToHex.h>
#include <lib/support/CHIPMem.h>
#include <lib/support/CHIPMemString.h>
#include <lib/support/CHIPPlatformMemory.h>

#include <system/SystemPacketBuffer.h>
#include <lib/support/verhoeff/Verhoeff.h>
```